source/CardGames.py

```diff
@@ -111,10 +112,34 @@ def showHand(self, printShort: bool = False):
           image = card.image[idx] if self.knownCards[i] else card.cardBack[idx]
           print(image, end="")
       print()
+  def High(self):
+    l =[]
+    l2 =[]
+    for c in self.hand:
+      l.append(str(c))
+    for c in l:
+      l2.append(int(c))
+    x = max(l2)
+    print(x)
 
   def clearHand(self):
     self.hand = []
     self.knownCards = []
+  def printCardsIfSet(self):
```

Comment:
Great use of the for loop when approaching this function. The code is intuitive and easy to follow. Good job!